src/components/toolbar/button.js

```diff
@@ -31,7 +31,7 @@ const ToolbarButton = ({
     </Button>
   )
 
-  return !tooltip ? (
+  return !tooltip || disabled ? (
```

Comment:
I don't think removing the tooltip is the solution, the problem is that it doesn't hide after showing it ( on firefox works as expected )

Reply:
On `brave` works `on-hover` but it doesnt hide for some reaseon

Reply:
I think another solution would be adding a boolean prop (disabled?) to the Tooltip (/components/Tooltip) component, which would control whether to pass the "visible" value from useTooltipState to the ReakitTooltip or to pass false on the "visible" prop

Reply:
Something like 
`
<ReakitTooltip
        {...tooltip}
        {...props}
        visible={!disabled && tooltip.visible}
        style={style}
        className={`tooltip-villain`}
        unstable_portal={false}
      >
`

Reply:
The problem is that the tooltip should still work on disabled components.
Please test it on firefox to see that it works as expected, but when switching to brave it fails to hide on `mouse-leave` or clicking outside etc...

Reply:
Oh and the tooltip seems to be showing up on disabled buttons on all the browsers I tested:
Chrome 78.0.3904.70
Firefox 71.0b4
Brave "Version 0.70.121 Chromium: 78.0.3904.70"

Reply:
Does it also hide ?

Reply:
Maybe is just a bug on linux ? :upside_down_face:

Reply:
Hmmm maybe, I'm on linux right now, I can test on windows when I get home.

It does hide as expected on brave and firefox, however on chrome it only hides if you move the mouse away from it and not into another button with a tooltip.

Reply:
I see from your `gif` it also fails on brave to behave consistently, I guess is an issue with `chromium` based browsers....